### DISCUSSION TITLE
Using Deepgram .NET SDK with .NET Framework 4.8

### DISCUSSION THREAD

**Discussion by ernestopedrosa** [deepgram/community#114] [General help]
Hello Community,
I'm building a project using .NET Framework 4.8 and I'm getting the following crash invoking deepgramlive.StartConnectionAsync(). Something related with adding User-Agent header by Deepgram code. Can somebody help me here?
Thanks!
E.
![deepgram crash](https://user-images.githubusercontent.com/11547062/229347619-2a5b8d84-3544-4aa0-886d-9203b39ff03c.png)

**Comment by briancbarrow** [comment]
Hi @ernestopedrosa,

This is likely an issue with you using .NET Framework 4.8. We only target the following with this SDK: 

- 6.0.0
- 5.0.0
- .NET Core 3.1
- .NET Standard 2.0

https://github.com/deepgram/deepgram-dotnet-sdk#targeted-frameworks
